mon-pix/app/services/url.js

```diff
@@ -48,6 +48,10 @@ export default class Url extends Service {
     return `https://pix.${this.currentDomain.getExtension()}`;
   }
 
+  get _showcaseWebsiteLinkText() {
+    return this.intl.t('navigation.showcase-homepage', { extension: this.currentDomain.getExtension() });
```

Comment:
Le nom de variable `extension` est perturbant car il est trop polysémique, et généralement on l'associe plus aux _« extensions de fichiers »_. Ne serait-il pas possible de renommer, dans ce contexte, `extension` en `tld` (pour [Top-level domain](https://en.wikipedia.org/wiki/Tld)) ?

Reply:
Je change par rapport à la traduction le `extension`en `tld`.
Pour le `this.currentDomain.getExtension()`, on est sur une méthode d'un autre service utilisé à un autre endroit. Je m'arrête là pour ce ticket.